datalad/distribution/get.py

```diff
@@ -273,15 +274,27 @@ def result_renderer_cmdline(res, args):
         if not isinstance(res, list):
             res = [res]
         if not len(res):
-            ui.message("Nothing was getted")
+            ui.message("Got nothing (we have it all already)")
             return
 
-        msg = linesep.join([
-            "{path} ... {suc}".format(
-                suc="ok." if item.get('success', False)
-                    else "failed. (%s)" % item.get('note', 'unknown reason'),
-                path=item.get('file'))
-            for item in res])
+        # provide summary
+        nsuccess = sum(item.get('success', False) for item in res)
+        nfailure = len(res) - nsuccess
+        msg = "Requested %d %s." % (len(res), single_or_plural("file", "files", len(res)))
```

Comment:
Also - nope. Currently it is not guaranteed, that the number of requested paths equals the number of entries in the results. See above - skipped ones are not included. Actually it is hard to decide what should count for being "requested" ... May be just "Tried to get ... files." ?